can/broadcastmanager.py

```diff
@@ -34,28 +35,42 @@ class CyclicSendTaskABC(CyclicTask):
     Message send task with defined period
     """
 
-    def __init__(self, message, period):
+    def __init__(self, messages, period):
         """
-        :param can.Message message: The message to be sent periodically.
-        :param float period: The rate in seconds at which to send the message.
+        :param List[can.Message] messages: The message to be sent periodically.
+        :param float period: The rate in seconds at which to send the messages.
         """
-        self.message = message
-        self.can_id = message.arbitration_id
-        self.arbitration_id = message.arbitration_id
+        if not isinstance(messages, list):
```

Comment:
`tuple` should also be supported. And unless we want to allow modification of `self.messages` sequence without using `ModifiableCyclicTaskABC.modify_data`, we should be using a `tuple` for `self.messages`

Reply:
Good point @pierreluctg!

I've changed the internal representation to a `tuple` type.